interface/login/login.php

```diff
@@ -29,267 +29,268 @@
-		        echo "<option value='".attr($iter['lang_id'])."'>" . text($iter['trans_lang_description']) . "</option>\n";
-		    }
-		}
+        function imsubmitted() {
+            <?php if (!empty($GLOBALS['restore_sessions'])) { ?>
+                // Delete the session cookie by setting its expiration date in the past.
+                // This forces the server to create a new session ID.
+                var olddate = new Date();
+                olddate.setFullYear(olddate.getFullYear() - 1);
+                document.cookie = '<?php echo session_name() . '=' . session_id() ?>; path=/; expires=' + olddate.toGMTString();
+            <?php } ?>
+            return false; //Currently the submit action is handled by the encrypt_form().
         }
-?>
-</select>
-</td></tr>
-<?php }} ?>
-
-<tr><td>&nbsp;</td><td>
-<input class="button large" type="submit" onClick="transmit_form()" value="<?php echo xla('Login');?>">
+    </script>
 
-</td></tr>
-<tr><td colspan='2' class='text' style='color:red'>
-<?php
-$ip=$_SERVER['REMOTE_ADDR'];
-?>
-</div>
-</td></tr>
-</table>
+</head>
+<body class="login">
+    <div class="container">
+        <form method="POST" id="loginForm"
+            action="../main/main_screen.php?auth=login&site=<?php echo attr($_SESSION['site_id']); ?>"
+            target="_top" name="login_form">
+            <div class="row">
+                <div class="col-sm-12">
+                    <div class="">
+                        <img class="img-responsive center-block" src="<?php echo $GLOBALS['webroot']?>/public/images/login-logo.svg" />
+                        <input type='hidden' name='new_login_session_management' value='1' />
 
-</div>
-<div style="clear: both;"> </div>
-<div class="version">
-<a  href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
-</div>
-</div>
+                        <?php
+                        // collect groups
+                        $res = sqlStatement("select distinct name from groups");
+                        for ($iter = 0;$row = sqlFetchArray($res);$iter++)
+	                          $result[$iter] = $row;
+                        if (count($result) == 1) {
+	                          $resvalue = $result[0]{"name"};
+	                          echo "<input type='hidden' name='authProvider' value='" . attr($resvalue) . "' />\n";
+                        }
+                        // collect default language id
+                        $res2 = sqlStatement("select * from lang_languages where lang_description = ?",array($GLOBALS['language_default']));
+                        for ($iter = 0;$row = sqlFetchArray($res2);$iter++)
+                            $result2[$iter] = $row;
+                        if (count($result2) == 1) {
+                            $defaultLangID = $result2[0]{"lang_id"};
```

Comment:
btw, I didn't change this code in this PR.
Not sure why it's using those brackets.